tests/pubsub_logging_test.py

```diff
@@ -41,6 +53,64 @@ def test_compat_urlsafe_b64encode(self):
         self.assertEqual(expected, result)
 
 
+class GetPubsubClientTest(unittest.TestCase):
+    """Tests for utils.get_pubsub_client function.
+
+    You have to set GOOGLE_APPLICATION_CREDENTIALS pointing to the
+    json file of the service account.
+    """
+    RETRY = 3
+
+    def test_get_pubsub_client_with_service_account(self):
+        """Tests the client obtained by the service account method."""
+        client = get_pubsub_client()
+        project = ('projects/%s'
+                   % os.environ.get(TEST_PROJECT_ENV, DEFAULT_TEST_PROJECT))
+        client.projects().topics().list(project=project).execute(
+            num_retries=self.RETRY)
+        # Providing an Http object this time.
+        client = get_pubsub_client(http=httplib2.Http())
+        self.assertIsNotNone(client)
+
+    def test_get_pubsub_client_with_scoped_credentials(self):
+        """Tests the client obtained by scoped credentials."""
+        credentials = GoogleCredentials.get_application_default()
+        credentials = credentials.create_scoped(PUBSUB_SCOPES)
+        client = get_pubsub_client(credentials=credentials)
+        self.assertIsNotNone(client)
+
+
+class CheckTopicTest(unittest.TestCase):
+    """Tests for utils.check_topic function.
+
+    You have to set GOOGLE_APPLICATION_CREDENTIALS pointing to the
+    json file of the service account.
+    """
+    RETRY = 3
+
+    def setUp(self):
+        self.client = get_pubsub_client()
+        self.project = os.environ.get(TEST_PROJECT_ENV, DEFAULT_TEST_PROJECT)
+        self.topic = 'projects/%s/topics/test-topic-%f' % (self.project,
+                                                           time.time())
+        self.nonexistence = 'projects/%s/topics/nonexistence' % self.project
+        self.client.projects().topics().create(
```

Comment:
Is it okay to call .create with a topic that's already been created? If so, great. If not, we should catch the error here.